tests/fixtures.py

```diff
@@ -1,14 +1,38 @@
 from datetime import datetime
+from pytz import UTC
+from uuid import uuid4
 
-from mlflow.entities import Metric, Param, RunTag
+from mlflow.entities import (
+    Experiment as MLExperiment,
```

Comment:
Elsewhere in this module we rename the Faculty classes and keep the MLflow classes at their original names. Can we do the same here?

Reply:
👍